ApiDemos/ApiDemos.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's unclear to me: what is this file for?

Reply:
Normally, it is used for keeping module configuration(created by default). However, it is unnecessary for the idea of the demo folder.